css/style.css

```diff
@@ -958,3 +1023,265 @@ input:focus {
     justify-content: space-between;
   }
 }
+
+/* Styling for works detail modal */
+
+#work-modal {
+  display: none;
+  overflow-y: scroll;
+  position: fixed;
+  z-index: 4;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(151, 148, 147, 0.8);
+}
+
+#work-modal .modal-content {
+  width: 343px;
+  background: #fff;
+  margin: 60px auto;
+  padding: 16px;
+}
+
+#work-modal .modal-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+#work-modal h2 {
+  color: #172b4d;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+#work-modal .tech-tags {
+  margin-top: 38px;
+  display: flex;
+}
+
+#work-modal .tech-tags a {
+  display: inline-block;
+  margin-top: 24px;
+  margin-right: 12px;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+}
+
+#work-modal .modal-main {
+  padding-top: 24px;
+  padding-bottom: 20px;
+}
+
+#work-modal .modal-main img {
+  /* height: 220px;
+  width: auto; */
```

Comment:
- [ ] Kindly remove unused blocks of code to keep the code-base clean and easy to navigate